lib/reverb/basics.h

```diff
@@ -104,11 +84,7 @@ T clamp (T value, T lower, T upper)
 	return value;
 }
 
-static inline float
-frandom()
-{
-	return (float) rand() / (float) RAND_MAX;
-}
+static inline float frandom() { return (float) random() / (float) RAND_MAX; }
```

Comment:
I think @ywwg probably changed this to ```rand``` since Windows doesn't have ```random```:

```
[CXX] lib\reverb\Reverb.cc
Reverb.cc
c:\projects\mixxx\lib\reverb\basics.h(36) : warning C4005: '_USE_MATH_DEFINES' : macro redefinition
        command-line arguments :  see previous definition of '_USE_MATH_DEFINES'
c:\projects\mixxx\lib\reverb\basics.h(87) : error C3861: 'random': identifier not found
scons: *** [lib\reverb\Reverb.obj] Error 2
scons: building terminated because of errors.
```